vhpidirect/arrays/intvector/vhdlpkgsized/main.c

```diff
@@ -0,0 +1,48 @@
+#include <stdio.h>
+#include <string.h>
+#include <malloc.h>
+
+extern int ghdl_main(char argc, char* argv[]);
+
+int main(int argc, char const *argv[])
+{
+	int vectorLength = 0;
```

Comment:
I feel that +20 lines is too much, since this is not the main feature that we want to show. I think we should accept a single `--grow-vec=3` or `--grow-vec 3` and multiply it with the size. More complex examples would fit in #14.

Reply:
I actually really want to move the main.c to #14...
We would do so after this is merged for sure. And it would just take the main.c file and the appropriate half of the run.sh file....